cdap-app-fabric/src/main/resources/tethering.json

```diff
@@ -0,0 +1,29 @@
+{
+  "configuration-groups": [
```

Comment:
will icon be added later? If so could you please file a jira to track it?

Reply:
created https://cdap.atlassian.net/browse/CDAP-18746